src/Client.ts

```diff
@@ -510,6 +506,21 @@ export default class Client {
                 return this.ban();
         }
     }
+
+    public setDevCheatsUsed(value: boolean) {
```

Comment:
Would you mind commenting these two methods? Just a simple header comment explaining generally what they do would be fine thank you.

Reply:
Done